server.js

```diff
@@ -1408,6 +1408,32 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// CDNJS version integration
+camp.route(/^\/cdnjs\/v\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var library = encodeURIComponent(match[1]);  // eg, "express" or "@user/express"
+  var format = match[2];
+  var apiUrl = 'https://api.cdnjs.com/libraries/' + library + '?fields=version';
+  var badgeData = getBadgeData('CDNJS', data);
```

Comment:
Updated! BTW, is it possible to add link to the generated markdown code by default? Like to link to the library on CDNJS, thanks.